doc/users/faq/troubleshooting_faq.rst

```diff
@@ -175,3 +175,27 @@ mailing list (please do not post git problems to the `users list
 Of course, you will want to clearly describe your problem, what you
 are expecting and what you are getting, but often a clean build and
 install will help.  See also :ref:`reporting-problems`.
+
+.. _pycharm-touble:
+
+PyCharm not recognizing Matplotlib after pip install
+====================================================
+
+First, make sure you install Matplotlib using the PyCharm terminal within the IDE.
+The command ``pip install matplotlib`` should install the library within the terminal 
+and installation can be verified checking the current Matplotlib version as shown below 
+or the reference earlier in the beginning of the `troubleshooting guide 
+<https://github.com/nelsost3/matplotlib/blob/main/doc/users/faq/troubleshooting_faq.rst#obtaining-matplotlib-version>`_.
+
+   >>> import matplotlib
+   >>> print(matplotlib.__version__)
+   '3.7.2'
+
+If unsuccessful with viewing the version, please attempt the following options as a potential 
+solution. The Python file within the directory should either be deleted and recreated with a new 
+name or edit the name of the Python file to a new name. This appears to be especially true if the
+file name is ``matplotlib.py`` within the working directory.
+
+An example would be to change ``matplotlib.py`` to the new file name of ``new_matplotlib.py`` and 
+attempting to check the version once more. If the version number now prints the issue has likely been 
+resolved and PyCharm should now be working correctly with the Matplotlib library.
```

Comment:
This is not specific to PyCharm, and I'm not sure we should be documenting how standard Python behaviour works. But if we do, it definitely doesn't belong in a PyCharm-specific section.

Reply:
I agree we should not add such an entry (this being true for all stdlib module names as well, not just matplotlib).